discord_bots/cogs/category.py

```diff
@@ -0,0 +1,337 @@
+            try:
+                category: Category = (
+                    session.query(Category)
+                    .filter(Category.name.ilike(category_name))
+                    .one()
+                )
+            except NoResultFound:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Could not find category **{category_name}**",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+            else:
+                category.is_rated = False
+                session.commit()
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Category **{category.name}** changed to **unrated**",
+                        colour=Colour.green(),
+                    )
+                )
+
+    @group.command(
+        name="setminleaderboardgames",
+        description="Set minimum number of games to be on the category leaderboard",
+    )
+    @app_commands.check(is_admin_app_command)
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(
+        category_name="Existing category",
+        min_num_games="Games required to show on the leaderboard",
+    )
+    async def setmingamesforleaderboard(
+        self,
+        interaction: Interaction,
+        category_name: str,
+        min_num_games: int,
+    ):
+        if min_num_games < 0:
+            await interaction.response.send_message(
+                embed=Embed(
+                    description=f"The minimum number of games must be non-negative",
+                    colour=Colour.red(),
+                ),
+                ephemeral=True,
+            )
+            return
+
+        session: SQLAlchemySession
+        with Session() as session:
+            category: Category | None = (
+                session.query(Category).filter(Category.name.ilike(category_name)).one()
+            )
+            if not category:
+                await interaction.response.send_message(
+                    embed=Embed(
+                        description=f"Could not find category **{category_name}**",
+                        colour=Colour.red(),
+                    ),
+                    ephemeral=True,
+                )
+                return
+            category.min_games_for_leaderboard = min_num_games
+            session.commit()
+        await interaction.response.send_message(
+            embed=Embed(
+                description=f"The minimum number of games required to appear on the leaderboard for category **{category.name}** is now **{min_num_games}**",
+                colour=Colour.green(),
+            )
+        )
+
+    @group.command(name="show", description="Show category details")
+    @app_commands.check(is_command_channel)
+    @app_commands.describe(category_name="Existing category")
```

Comment:
`app_copmmands.rename` `category_name` to `category`

Reply:
Added a rename decorator.
Did so for the remaining functions in `category.py`